thirdparty/build-thirdparty.sh

```diff
@@ -295,7 +295,7 @@ build_gtest() {
     check_if_source_exist $GTEST_SOURCE
 
     cd $TP_SOURCE_DIR/$GTEST_SOURCE
-    mkdir build -p && cd build
+    mkdir -p bld && cd bld
```

Comment:
I think `build` is easier to understand than `bld`

Reply:
Projects that support build by `bazel` will contain a file named `BUILD`. Since the filesystem of Mac OS is `not` case sensitive, you cannot create a directory named `build` for those projects.
```
double-conversion-3.1.1/BUILD
re2-2017-05-01/BUILD
protobuf-3.5.1/BUILD
gflags-2.2.0/BUILD
incubator-brpc-0.9.5/BUILD
brotli-1.0.7/BUILD
flatbuffers-1.10.0/BUILD
mysql-server-mysql-5.7.18/BUILD
cctz-2.3/BUILD
glog-0.4.0/BUILD
```

Reply:
Thanks，I see.
Better to add comments about this reason. And is doris_build better?

Reply:
updated